lib/helpers/Profile/profileActiveHelpers.js

```diff
@@ -7,6 +7,14 @@ if (Meteor.isClient){
 
 			var id = Meteor.userId();
 			return Listing.find({ $and:[ { creator_id: Meteor.userId(), status: "Accepted" }, { offerAccepted: Offer.find({_id: this.offerAccepted})  } ] });
+		},
+		// Check to see if results exceeds 0
+		defaultCheck: function () {
+			var offerid = this.offerAccepted;
+
+			var id = Meteor.userId();
+			var results = Listing.find({ $and:[ { creator_id: Meteor.userId(), status: "Accepted" }, { offerAccepted: Offer.find({_id: this.offerAccepted})  } ] });
```

Comment:
Line is too long.<br>Identifier 'creator_id' is not in camel case.<br>'Listing' is not defined.<br>'Meteor' is not defined.<br>'Offer' is not defined.